geo-types/src/rect.rs

```diff
@@ -12,6 +12,35 @@ where
 }
 
 impl<T: CoordinateType> Rect<T> {
+    /// Creates a new rectangle.
+    ///
+    /// # Examples
```

Comment:
Would you mind adding a `# Panics` section above this documenting how this function can panic? Thanks!

Reply:
Just update the doc to mention the panic conditions.